Resources/Prototypes/Entities/Objects/Tools/cable_coils.yml

```
@@ -33,13 +33,13 @@
   #1 Ointment = -50 damage of those types
   #1 Cable ~= -50 (-49.8) damage of those types
   - type: Healing
-    delay: 0.6
+    delay: 0.7
     damageContainers:
     - Silicon
     damage:
       types: # these are all split across multiple types
```

yamllint

[warning] too few spaces before comment

(comments)